backend/src/prompts/README.md

```diff
@@ -0,0 +1,10 @@
+To get started, set your OPENAI_API_KEY environment variable, or other required keys for the providers you selected.
+
+install promptfoo by running `npx install promptfoo`
+
+Next, edit promptfooconfig.yaml.
+
+Then run `promptfoo eval` to run promptfooconfig.yaml
+ro run a specific file for example `generate_message_suggestions_config.yaml` run `promptfoo eval -c generate_message_suggestions_config.yaml`
+
+Afterwards, you can view the results by running `promptfoo view`
```

Comment:
Should these live under `backend/tests/prompts` instead? ... to be honest, I don't know what the tests in there are doing, or how they are supposed to be run...

It's becoming a bit of a mix...

Reply:
The tests we have under "prompts" already aren't doing a great deal in my opinion. Looks to me like it's just duplicating the code.

e.g. if someone was to make a change to the prompt file, the test would fail and then they would update the prompt exactly the same in the prompts file. I suppose it might alert you if you changed a prompt that you didn't realise was being nested inside another prompt somewhere else?

I would be tempted to scrap all the tests under `/backend/tests/prompts` and replace it with these...

Reply:
That being said, the tests in that folder are python based, and this is an npm based test. So maybe mixing them isn't a good idea. Possibly we could have it under `/backend/promptfoo`, I'm not sure...

Reply:
Because of how python does import you can't do that - at least I couldn't figure out how to 
the problem line is `from prompting import PromptEngine` from get_generate_message_suggestions_prompt.py

can show you if you want

Reply:
I've made some updates to fix some of these issues